src/core/basics.ml

```diff
@@ -4,15 +4,9 @@ open Extra
 open Timed
 open Terms
 
-(** Sets and maps of variables. *)
-module Var =
-  struct
-    type t = term Bindlib.var
-    let compare = Bindlib.compare_vars
-  end
-
-module VarSet = Set.Make(Var)
-module VarMap = Map.Make(Var)
+(** Create an array of fresh variables [|x1;..;xn|]. *)
+let fresh_vars : int -> tvar array = fun n ->
+  Bindlib.new_mvar mkfree (Array.init n (Printf.sprintf "x%i"))
```

Comment:
I'm not sure how I feel about this function. It is generally a good idea to preserve some kind of name, so that kind of thing should only be done as a last resort. Also, generic names should probably be avoided here. I would rather implement it as:
```ocaml
  Bindlib.new_mvar mkfree (Array.make n "_generated_var_" (* or "_var_" *)) 
```
Note that `Bindlib` should take care of numbering to avoid name clashes. This will happen at the time of binding.